src/accumulator/util.rs

```diff
@@ -12,27 +12,62 @@ pub fn is_root_position(position: u64, num_leaves: u64, forest_rows: u8) -> bool
     return root_present && root_pos == position;
 }
 
-// extractTwins is a optimization for batched deletions. It checks if the nodes
```

Comment:
For future reference:

Extract_twins is really being removed and then remove_bit, calc_next_pos, and detwin is being added. But if you do this in a single step, `git diff` thinks that extract_twins is being modified to be remove_bit.

It's fine here because this is a small commit but for bigger commits and for bigger projects, it'll be a lot more helpful during reviewing if you did these steps separately.